test/runtests.jl

```diff
@@ -101,6 +101,66 @@ else  # run HiGHS tests
         @test r["ElectricStorage"]["size_kwh"] ≈ 83.3 atol=0.1
     end
 
+    @testset "Solar and ElectricStorage w/ zero ElectricStorage cost constants" begin
+        m1 = Model(optimizer_with_attributes(HiGHS.Optimizer, "output_flag" => false, "log_to_console" => false))
+        m2 = Model(optimizer_with_attributes(HiGHS.Optimizer, "output_flag" => false, "log_to_console" => false))
+        d = JSON.parsefile("./scenarios/pv_storage.json");
+        
+        d["ElectricStorage"]["installed_cost_constant"] = 0
+        d["ElectricStorage"]["replace_cost_constant"] = 0
+        d["ElectricStorage"]["cost_constant_replacement_year"] = 10
+    
+        d["Settings"] = Dict{Any,Any}("add_soc_incentive" => false)
+        s = Scenario(d)
+        inputs = REoptInputs(s)
+        results = run_reopt([m1,m2], inputs)
+        
+        UpfrontCosts_NoIncentive = (results["ElectricStorage"]["size_kw"]*d["ElectricStorage"]["installed_cost_per_kw"] ) +
+                                   (results["ElectricStorage"]["size_kwh"]*d["ElectricStorage"]["installed_cost_per_kwh"]) + 
+                                   d["ElectricStorage"]["installed_cost_constant"] +
+                                   (results["PV"]["size_kw"]*d["PV"]["installed_cost_per_kw"])
+    
+        @test results["PV"]["size_kw"] ≈ 216.6667 atol=0.01 
```

Comment:
@toddleif just because so many other things could affect both PV sizing and LCC, it could make sense to avoid testing for these specific values here. That way, we don't have to update this test every time other changes are made that don't relate to the BESS cost constant value.

Reply:
That's a great point, thank you. I removed a number of the values to test.